Automattic/wp-calypso
# Remove unused site editor overrides.

#### Changes proposed in this Pull Request

This removes our `wpcom-site-editor` overrides in the ETK plugin. The only override here is an outdated and unused navigation toggle override. 

While unused, removal seems time sensitive as @zaguiini verified that Gutenberg `trunk` branch now uses these class names for this override once again which would cause this override to work again (we don't want that) with the next Gutenberg upgrade.

#### Testing instructions



* Patch this ETK build to your sandbox.
* Verify the site editor continues to work as expected.



Related to https://github.com/Automattic/wp-calypso/issues/57834
